src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -1,3 +1,4 @@
+import { clipboard } from 'electron';
```

Comment:
Why do you use an electron-specific `clipboard` API instead of the standard `clipboard` Web API?

Reply:
Because I didn't know that existed lol

Reply:
I even linked to it in my previous comment...

Reply:
Whoops. I think I just just saw clipboard.write, tried to use it with electron, couldn't, and then used writeBuffer. Should have taken a look at the link instead, sorry.

Anyway, is there a benefit to using the web one instead of the electron one?

Reply:
Mostly that it's more commonly used. Web APIs have the advantage that a lot of people use them, so they are more stable and better documented. Also, Electron started before the web clipboard API was introduced, so they might remove the Electron API at some point.